font-patcher

```diff
@@ -609,6 +615,7 @@ class font_patcher:
             {'Enabled': self.args.octicons,             'Name': "Octicons",                'Filename': "octicons.ttf",                                   'Exact': self.octiconsExactEncodingPosition,  'SymStart': 0X26A1, 'SymEnd': 0X26A1, 'SrcStart': None,   'SrcEnd': None,   'ScaleGlyph': OCTI_SCALE_LIST,  'Attributes': SYM_ATTR_DEFAULT},  # Zap
             {'Enabled': self.args.octicons,             'Name': "Octicons",                'Filename': "octicons.ttf",                                   'Exact': self.octiconsExactEncodingPosition,  'SymStart': 0xF27C, 'SymEnd': 0xF27C, 'SrcStart': 0xF4A9, 'SrcEnd': 0xF4A9, 'ScaleGlyph': OCTI_SCALE_LIST,  'Attributes': SYM_ATTR_DEFAULT},  # Desktop
             {'Enabled': self.args.codicons,             'Name': "Codicons",                'Filename': "codicons/codicon.ttf",                           'Exact': True,                                'SymStart': 0xEA60, 'SymEnd': 0xEBEB, 'SrcStart': None,   'SrcEnd': None,   'ScaleGlyph': None,             'Attributes': SYM_ATTR_DEFAULT},
+            {'Enabled': self.args.braille,              'Name': "Braille",                 'Filename': "UBraille.ttf",                     'Exact': False,                               'SymStart': 0x2800, 'SymEnd': 0x28FF, 'SrcStart': None,   'SrcEnd': None,   'ScaleGlyph': None,             'Attributes': SYM_ATTR_DEFAULT},
```

Comment:
I guess we need a ScaleGlyph here, to have a working `mono` version?
About the ScaleGlyph ... without one the patching will maximize the size of each glyph individually to fill the 'cell' completely. But with Braille you want to scale all glyphs _identical_ to keep the relative size/position differences intact.

Further the `'Attributes': SYM_ATTR_DEFAULT` seems an odd choice, as that means `'align': 'c', 'valign': 'c', 'stretch': 'pa'`. That does only make sense if a `ScaleGlyph` is active? And maybe we want a `careful` here?

Also maybe correct the whitespaces, the vertical alignment (nothing I like normally, but we have it here) is broken as you see from the diff. Maybe a tab instead of spaces?

Reply:
Thank you for looking through the code. I amended the vertical alignment and moved the issue number to a line of its own in the commit message.

I'm afraid I don't know enough about fonts to touch the attributes, so (everyone) feel free to improve things.

Reply:
Here the scale effect for illustration:

![image](https://user-images.githubusercontent.com/16012374/171393356-005cb24c-1184-4d4f-a0d2-5c2bf95b5a54.png)

Lets fix it...

Reply:
Fixed in additional commit:

![image](https://user-images.githubusercontent.com/16012374/171397075-04339049-9b7a-407d-8230-b43788ca6deb.png)

Reply:
After some thought I believe we do not want `careful` here, at least not a partial one.
If there is any glyph in the range it will not match our new glyphs in style and size.
I guess best is to not be careful here.

Reply:
OH NOO, it is not working, the scaling is uniform but the centering is individual? Now we have a lot centered single dots!

Examining...

Reply:
Well, `ScaleGlyph` can not (by design) enforce consistent shifting/centering.

This needs some thought.